### DISCUSSION TITLE
How to achieve showChildLevel feature, similar to showParentLevel={false}

### DISCUSSION THREAD

**Discussion by jhaankit** [algolia/instantsearch#5280] [Ideas]
Algolia provides a feature showParentLevel={boolen}.
How to achieve similar to this feature where a user can only see the children of selected facet values?

An example use case:
Step1: We have facet values at level-0 as below
```
-A0
-B0
-C0
```

Step2: On selection of B0, only the tree of B0 should display (assume B0 has S1, T1, U1 facet values in it)
```
-B0
   -S1
   -T1
   -U1
```

Step3: On selection of S1, the same level should display as there is no child to S1.
```
-S1
-T1
-U1
```

**Comment by Haroenv** [comment]
If a user would like to "remove" that applied filter, how could they do it when the filter which is set is no longer displaying?

**Reply by jhaankit** [reply]
With the help of back button, as shown in the pic
<img width="577" alt="image" src="https://user-images.githubusercontent.com/11736703/161119661-83592e89-f429-4bd3-9f79-8b1efcd96a3b.png">
